packages/wallets/package.json

```diff
@@ -15,7 +15,7 @@
     },
     "dependencies": {
         "@deriv-com/analytics": "1.11.0",
-        "@deriv-com/translations": "1.3.4",
+        "@deriv-com/translations": "1.3.5",
```

Comment:
Updated `deriv-com//translations` version to incorporate [this fix](https://github.com/deriv-com/translations/pull/50) to remove deprecated tag